model/client4.go

```diff
@@ -2807,8 +2807,12 @@ func (c *Client4) ListCommands(teamId string, customOnly bool) ([]*Command, *Res
 }
 
 // ExecuteCommand executes a given command.
-func (c *Client4) ExecuteCommand(channelId, command string) (*CommandResponse, *Response) {
-	commandArgs := &CommandArgs{ChannelId: channelId, Command: command}
+func (c *Client4) ExecuteCommand(teamId, channelId, command string) (*CommandResponse, *Response) {
```

Comment:
This change isn't really necessary is it? I think we aim to keep the ClientX APIs stable.

Reply:
Yes, sorry @MusikPolice I meant to mention that but forgot after we went down the rabbit hole. We should create a new function `ExecuteCommandWithTeamId` if we need to specify the team ID